functions/two_sum.py

```diff
@@ -1,2 +1,5 @@
 def twoSum(n, t):
-    pass
+    for i in range(len(n)):
+        for j in range(i + 1, len(n)):
```

Comment:
could refactor to more efficient solution than O(n^2)

Reply:
I agree ^